signal/cmd/root.go

```diff
@@ -15,14 +15,18 @@ const (
 )
 
 var (
+	// will be replaced with the release version when using goreleaser
+	version = "development"
```

Comment:
we moved away from the global var to keep version in a single package `system`, have a look at https://github.com/netbirdio/netbird/blob/main/client/cmd/version.go, main benefits of this approach is that we can use the system.NetbirdVersion() call in other places and keep consistency

Reply:
That package is inside in the client tree. Should I move out the version logic to outer, to the root?